WordPress/WordPressTest/NSStringHelpersTest.m

```diff
@@ -116,4 +116,23 @@ - (void)testIsWordPressComPathWithInvalidProtocols
     }
 }
 
+- (void)testWordCount
+{
+    NSString *testEmptyPhrase = @"";
+    XCTAssert([testEmptyPhrase wordCount] == 0, @"Word count should be zero on a empty string");
+    
+    NSString *testPhraseEnglish = @"The lazy fox jumped over the fence.";
```

Comment:
I tend to strive for single asserts in a test - or a number of asserts to check the logical conclusion of an operation. In my mind unit tests should really only fail for exactly one reason. I'm cool with leaving this be since we really have no consensus on unit testing rules :+1: